librz/core/cmd_analysis.c

```diff
@@ -3265,45 +3229,6 @@ RZ_IPI int rz_cmd_analysis_fcn(void *data, const char *input) {
 			rz_core_analysis_bbs_asciiart(core, fcn);
 			break;
 		}
-		case 'i': // "afbi"
```

Comment:
because these are already converted to newshell and even though i'd like to keep oldshell code for a little longer, keeping this would require to mess up a bit too much with the code to be able to call the function with the new RzCmdOutputState structure correctly set up. So I've preferred to delete it. Any objection?

Reply:
No objections. I am all for removing the oldshell code wherever appropriate even right now.